Modules/Server/Sources/PocketCastsServer/Public/Search/PodcastSearchTask.swift

```diff
@@ -2,15 +2,28 @@ import Foundation
 import PocketCastsDataModel
 
 struct PodcastsSearchEnvelope: Decodable {
+    let status: String
```

Comment:
Are we sure this is a `String` that will always be there? In `PodcastSearchResponse` this seems to be optional, it might cause decoding issues although it seems everything is working as expected.

Reply:
From my tests, there always there, is the an API spec that we could double check? or maybe a link to the server code?

Reply:
@geekygecko confirmed that we always have a status value.